ipinfo/ipinfo.go

```diff
@@ -1,144 +1,11 @@
 //go:generate go run gen-fields.go
 
-package ipinfo // import "github.com/ipinfo/go-ipinfo/ipinfo"
+package ipinfo
 
-import (
-	"bytes"
-	"net"
-	"strings"
-)
-
-var c *Client
+// Global, default client available to the user via `ipinfo.DefaultClient`.
```

Comment:
golint: comment on exported var DefaultClient should be of the form "DefaultClient ..."